lib/src/Uncertainty/Distribution/PosteriorDistribution.cxx

```diff
@@ -107,7 +107,7 @@ Point PosteriorDistribution::computeLikelihood(const Point & theta) const
 Scalar PosteriorDistribution::computeLogLikelihood(const Point & theta) const
```

Comment:
Here theta should be renamed y to match the notations of the reference document. Theta is the image of y by the link function, and is also the name of the parameter of the conditioned distribution.

Reply:
Other point: in computePDF(), we compute the log-pdf before to take its exponential. It would be more logical to implement computeLogPDF() then to call it from computePDF()

Reply:
done, theta=f(y)